web/src/lib/domain-logic/essentialQuestions.ts

```diff
@@ -0,0 +1,155 @@
+import { EssentialQuestionsProps, ProfileLabelField } from "@/lib/types/types";
+import {
+  emptyIndustrySpecificData,
+  Industry,
+  IndustrySpecificData,
+  LookupIndustryById,
+  ProfileData,
+} from "@businessnjgovnavigator/shared";
+
+type EssentialQuestionFunction = (industry: Industry) => boolean;
+interface EssentialQuestion
+  extends Omit<EssentialQuestionsProps<ProfileData[keyof ProfileData]>, "fieldName" | "choices"> {
+  fieldName: keyof IndustrySpecificData;
+  validationFunc: EssentialQuestionFunction;
+  reset: boolean;
```

Comment:
"reset" here reads as an action, like a function, but it's actually a boolean and I think it'd be helpful for us to understand if it was named more clearly. I'm trying to guess what this field represents based on the code below and I'm guessing something like `shouldBeResetWhenIndustryChanges`?